tests/conftest.py

```diff
@@ -36,6 +36,8 @@
 
 from tests.common.connections.console_host import ConsoleHost
 
+WAIT_FOR_COUNTERS_TIMEOUT = 190
```

Comment:
>WAIT_FOR_COUNTERS_TIMEOUT [](http://example.com/codeflow?start=0&length=25)

How is the constant chosen? If we change image, do we need to update the contant? If it is calculated by other constant in the image, please add code reference in comments.

Reply:
The constant is chosen by the delay which can be found here:
https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-orchagent/enable_counters.py
The value is the same on all branches.